### PR TITLE
Add share button for safety numbers

### LINKED ISSUE #5798: Add possibility to share safety numbers
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

---
### Feature Request

Besides #5795 (copyable safety numbers) it would be nice to have a share button in the action bar of `VerifyIdentityActivity`. This would offer a faster work flow for sharing the safety number compared to "copy & paste".

### PR DESCRIPTION
**Pull request by FeuRenard**
### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * Sony Xperia U, Android 4.4.4 (CyanogenMod)
 * AVD Nexus 5X, Android 6.0
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in the commit message of my first commit

----------

### Description
This PR adds a share button to `VerifyIdentityActivity`. Actually a string is shared which consists of:

- short description "Our Signal safety numbers"
- the safety numbers, each block separated by one space and grouped by four blocks per line. So they are formatted like in-app.

I was not sure about two things:

- add a line break after the safety numbers? Removed the line break with 70cee84. But this commit can be dropped and the line break will be there again. Maybe people sharing their safety numbers want to add further explanation...
- The short description could also be placed in the subject of the shared intent (`EXTRA_SUBJECT`). But not all apps accepting shared text make use of this subject. For those apps there would be no description provided along the numbers. Therefore I added 7ff6cde. This commit can also be dropped.

Fixes #5798
// FREEBIE

### Screenshots
device-2016-11-12-191809
device-2016-11-12-191840
device-2016-11-12-191906

### CODE REVIEW

**Comment by RiseT** [comment]
Without reading your corresponding explanations, I've instantly thought "There needs to be a line break after Safety Numbers". So yes - there definitely should be a line break in my opinion.

**Comment by RiseT** [comment]
Regarding the subject: I'd say put the description text there in addition to putting it in the text body. If an app doesn't support this, the description is still in the body.

File: src/org/thoughtcrime/securesms/VerifyIdentityActivity.java [hunk 1]
```diff
@@ -294,6 +333,10 @@ public void setClickListener(View.OnClickListener listener) {
       this.clickListener = listener;
     }
 
+    TextView[] getCodes() {
```
Comment (moxie0) [inline src/org/thoughtcrime/securesms/VerifyIdentityActivity.java]: A fragment should not return its views back to the parent activity. If anything you want to retrieve the Fingerprint object.

Comment (FeuRenard) [inline src/org/thoughtcrime/securesms/VerifyIdentityActivity.java]: Ok, I will modify this tomorrow.

**Pull request closed without merging**